src/osd/OSD.h

```diff
@@ -1529,7 +1528,6 @@ class OSD : public Dispatcher,
     assert(osd_lock.is_locked());
     finished.splice(finished.end(), ls);
   }
-  void do_waiters();
```

Comment:
If this machinery is dead, we need to remove the finished and take_waiters() members as well. But we're still pushing stuff on in OSD::activate_map(), from when OSD::handle_pg_create() pushes it on via require_same_or_newer_map(). So I don't think we can kill it quite yet.

Reply:
yeah, teh legacy pg create handler is still using it!

Reply:
So that means we still need to do_waiters() then.